mantle/kola/harness.go

```diff
@@ -1244,11 +1245,22 @@ func runTest(h *harness.H, t *register.Test, pltfrm string, flight platform.Flig
 			AppendKernelArgs: t.AppendKernelArgs,
 		}
 
-		_, err = platform.NewMachines(c, userdata, t.ClusterSize, options)
+		// Providers sometimes fail to bring up a machine within a
+		// reasonable time frame. Let's try twice and then bail if
+		// it doesn't work.
+		err := util.Retry(2, 1*time.Second, func() error {
+			var err error
+			_, err = platform.NewMachines(c, userdata, t.ClusterSize, options)
```

Comment:
Minor/optional: I think this could be

```suggestion
			_, err := platform.NewMachines(c, userdata, t.ClusterSize, options)
```

Reply:
yeah I was trying to be explicit that it's not the same `err` as above.